src/components/Home/index.jsx

```diff
@@ -308,16 +555,51 @@ export default class Home extends React.Component {
     this.setState({ SCROLL_POS: (st / (sh - h)) * 100 });
   }
 
+  scrollRight() {
+    if (this.state.FAQ_STATE === 'General') {
+      this.setState({ FAQ_STATE: 'Before', PREV_FAQ: 'General' });
+    }
+    if (this.state.FAQ_STATE === 'Before') {
+      this.setState({ FAQ_STATE: 'During', PREV_FAQ: 'Before' });
+    }
+  }
```

Comment:
This (and the following scrollLeft) looks like it could be simplified using an array ( `['General', 'Before', 'During']` ) or even just reusing CLICKABLES.